app/src/main/java/com/x8bit/bitwarden/ui/vault/feature/addedit/util/CipherViewExtensions.kt

```diff
@@ -134,11 +134,16 @@ fun VaultAddEditState.ViewState.appendFolderAndOwnerData(
                 ),
                 selectedOwnerId = activeAccount.toSelectedOwnerId(
                     cipherView = currentContentState.common.originalCipher,
-                ),
+                )
+                    ?: collectionViewList.firstOrNull {
```

Comment:
Since this is now chaining, can we clean up the formatting here:
```kotlin
selectedOwnerId = activeAccount
    .toSelectedOwnerId(cipherView = currentContentState.common.originalCipher)
    ?: collectionViewList
        .firstOrNull { it.id == currentContentState.common.selectedCollectionId }
        ?.organizationId,
```